wiki/en/en-Compiling.md

```diff
@@ -62,12 +62,22 @@ Will build the file and make it available in `./Release/Jamulus.app`
 ## Windows
 You don't need to compile Jamulus for Windows since we provide official binaries, but if you really want to, follow the instructions below.
 
-**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio, ASIO development files
+### Required Software
+- A compiler like [Visual Studio](https://visualstudio.microsoft.com)
+- [Qt](https://www.qt.io/download)
+    - Use the free GPLv2 license for Open Source development
+    - To determine the Qt version you need, check [qt-installer-windows.qs](https://github.com/corrados/jamulus/blob/master/windows/qt-installer-windows.qs): under INSTALL_COMPONENTS you will see `qt.qt5.[version]`, e.g., 5123 means version 5.12.3.
+    - Select Components during installation: Expand the Qt section, find the matching version, e.g., **Qt 5.12.3**, and add the compiler components for your compiler, e.g., `MSVC 2017 32-bit/64-bit` for Visual Studio 2019
```

Comment:
Is it really MSVC 2017 for Visual Studio 2019?

Reply:
Yes, it is working for me.

Reply:
Ok. Not exactly sure what MSVC 2017 is then. I thought it was the compiler and there might be compatibility issues if you choose MSVC 2017 but run Visual Studio 2019.